src/components/Calculator.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import './styles.css';
+
+function Calculator() {
+  const [input, setInput] = useState('');
+  const calcBtns = [];
+  [7, 8, 9, 4, 5, 6, 1, 2, 3, 0, '.'].forEach((item) => {
+    calcBtns.push(
+      <button
+        type="button"
+        onClick={(e) => {
+          setInput(input + e.target.value);
+        }}
+        value={item}
+        key={item}
+      >
+        {' '}
+        {item}
+      </button>,
+    );
+  });
+
+  return (
+    <div className="wrapper">
+      {' '}
+      <div className="show-input">{input}</div>
+      <div className="digits flex">{calcBtns}</div>
+      <div className="modifiers subgrid">
+
```

Comment:
- [ ] Please ensure that the calculator component has at least one child component. I recommend creating a `<Button/>` child component to meet the project requirements. :pray: